docs/releasing/releasing.md

```diff
@@ -36,7 +37,7 @@ Follow the steps below to start the GitHub actions workflow which build the bina
     ```
 
 ## Verifying a draft release
-Only a member of the research group logged into GitHub can see draft releases.
+Only a member of the [research group Energy Transition](https://github.com/energietransitie) logged into GitHub can see draft release.
```

Comment:
See previous comment.

Reply:
fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/commit/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5